packages/esm-ohri-pmtct-app/src/dashboard.meta.tsx

```diff
@@ -33,3 +35,24 @@ export const mchFolderMeta = {
   slotName: 'mch-slot',
   isExpanded: false,
 };
+
+// Clinical Dashboards
+// export const motherChildHealthFolderDashboardMeta = {
+//   slot: 'ohri-mother-child-health-dashboard-slot',
+//   config: { columns: 1, type: 'grid', icon: ParentChild },
+//   isFolder: true,
+//   title: 'Mother Child Health',
+// };
```

Comment:
clean up this commented code if not needed

Reply:
@pirupius Done